CHANGELOG.md

```diff
@@ -5,6 +5,33 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](https://semver.org/).
 
+## [v0.33.1] - 2023-04-25
+
+### Added
+
+- feat: Add behavior test for read_with_if_match & stat_with_if_match (#2088)
+- feat(tests): Add fuzz test for writer without content length (#2100)
+- feat: add if_none_match support for obs (#2103)
+- feat(services/oss): Add server side encryption support for oss (#2092)
+- feat(core): update errorKind `PreconditionFailed` to `ConditionNotMatch` (#2104)
```

Comment:
May notice here in the upgrade.md

Reply:
We will update `updatede.md` when we have breaking changes.